docs/_includes/home-lhn.html

```diff
@@ -0,0 +1,13 @@
+<ul class="lhn-items">
+    <li class="home-link">
+        <a href="/index.html">Home</a>
+    </li>
+    <li class="icon-with-link">
+        <i class="fa-solid fa-angle-right icon"></i>
+        <a href="/hubs/send-money">Send money</a>
+    </li>
+    <li class="icon-with-link">
+        <i class="fa-solid fa-angle-right icon"></i>
```

Comment:
I was expecting that clicking on the right caret icon would allow me to navigate the docs as well. What do you think?

![Screen Shot 2022-07-11 at 4 55 42 PM](https://user-images.githubusercontent.com/12268372/178269945-c4fe1b0a-a132-4090-93ad-08744c16d35b.png)

Reply:
Good point, making the caret icons also clickable and adding the `cursor: pointer` value to them